pkg/os/launchd/launchd_darwin.go

```diff
@@ -17,36 +17,49 @@ import (
 )
 
 const (
-	plistTemplate = `<?xml version='1.0' encoding='UTF-8'?>
-	<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-	<plist version='1.0'>
+	daemonPlistTemplate = `<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version='1.0'>
+<dict>
+	<key>Label</key>
+	<string>{{ .Label }}</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>{{ .ExecutablePath }}</string>
+	{{ range .Args }}
+		<string>{{ . }}</string>
+	{{ end }}
+	</array>
+	<key>StandardOutPath</key>
+	<string>{{ .StdOutFilePath }}</string>
+	<key>StandardErrorPath</key>
+	<string>{{ .StdErrFilePath }}</string>
+	<key>Sockets</key>
+	<dict>
+		<key>Listeners</key>
 		<dict>
-			<key>Label</key>
-			<string>{{ .Label }}</string>
-			<key>ProgramArguments</key>
-			<array>
-				<string>{{ .ExecutablePath }}</string>
-			{{ range .Args }}
-				<string>{{ . }}</string>
-			{{ end }}
-			</array>
-			<key>StandardOutPath</key>
-			<string>{{ .StdOutFilePath }}</string>
-			<key>Disabled</key>
-			<false/>
-			<key>RunAtLoad</key>
-			<true/>
-			<key>ProcessType</key>
-			<string>Interactive</string>
+			<key>SockFamily</key>
+			<string>Unix</string>
+			<key>SockPathName</key>
+			<string>{{ .SocketPath }}</string>
+			<key>SockPathMode</key>
+			<integer>493</integer>
+			<key>SockType</key>
+			<string>stream</string>
```

Comment:
Isn't this configuring socket activation for the daemon, which we don't want to use yet?